partcad/src/partcad/sketch.py

```
@@ -16,16 +16,13 @@
+    def __init__(self, project_name: str, config: object = {}) -> None:
+        super().__init__(project_name, config)
 
-        self.kind = "sketches"
+        self.kind = "sketch"
 
-    async def get_shape(self):
-        async with self.locked():
-            if self.shape is None:
-                self.shape = await pc_thread.run_async(self.instantiate, self)
-            return self.shape
+    async def get_shape(self, ctx):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_shape`

(ANN201)